### DISCUSSION TITLE
Passthrought class on Password

### DISCUSSION THREAD

**Discussion by andreas83** [primefaces/community#2007] [PrimeVue] [answered]
To me it looks like the class variable does not pass throuh while using the <Password> field.

It works only on root element but not on input  field itself:

`<Password :pt="{input:{class:'border-2'}}" />`

**Comment by tugcekucukoglu** [comment, marked as answer]
Password uses InputText internally. so the implementation should be like this:

```
<Password :pt="{input: {root: 'classname'}}"
```

**Reply by andreas83** [reply]
thank you :)
